src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py

```diff
@@ -60,7 +60,7 @@ def secure_copy(user, host, src, dest, key_filename=None, allow_agent=True):
     ssh = paramiko.SSHClient()
     proxy = None
     ssh_config_file = os.path.expanduser("~/.ssh/config")
-    if os.path.exists(ssh_config_file):
+    if os.path.exists(ssh_config_file) and os.path.isfile(ssh_config_file):
```

Comment:
I don't think you need the old condition of `os.path.exists(ssh_config_file)` now.